packages/react-components/react-storybook/src/index.ts

```diff
@@ -1 +1,8 @@
-export { withFluentProvider, withStrictMode } from './decorators/index';
+export {
+  withFluentProvider,
+  withStrictMode,
+  withFluentVrTestVariants,
+  DARK_MODE,
+  HIGH_CONTRAST,
+  RTL,
+} from './decorators/index';
```

Comment:
no need for using `index` - this was need in v8 and lower for AMD consumer. this package doesn't have to do that.

we should update lint rules for this - TODO(ME): create issue

```suggestion
} from './decorators';
```